packages/web/src/scss/tools/_dictionaries.scss

```
@@ -0,0 +1,17 @@
+@use 'sass:map';
+@use '@tokens' as tokens;
+@use '../settings/dictionaries';
+
+@mixin generate-colors-variant($properties, $variant-value) {
```

If we don't expect this mixin to be reusable outside this module, it is better to be private:

```suggestion
@mixin _generate-colors-variant($properties, $variant-value) {
```